tsconfig.json

```diff
@@ -13,7 +13,16 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "jsx": "preserve",
-    "incremental": true
+    "incremental": true,
+    "baseUrl": "./",
+    "paths": {
+      "@components/*": ["components/*"],
+      "@api/*": ["pages/api/*"],
+      "@stitches": ["stitches.config.ts"],
+      "@pages/*": ["pages/*"],
+      "@public/*": ["public/*"],
+      "@content/*": ["content/*"]
```

Comment:
Still getting build failures on vercel: 
![image](https://user-images.githubusercontent.com/7143571/149019536-0118884d-bcc8-480e-a141-8e39ee2d8e7a.png)

Are these namespaces meant to make it easier to move files around? if so, it's a cool idea, but since we're still seeing build issues, let's keep the changes scoped to the Nav component

Reply:
yes just makes it easier to import instead of having to figure out the correct ../../../, and agreed on keeping the scope but im not sure how to undo commits.... but this build failure looks like its just because these imports didnt get changed to lowercase layout in the original commits